docs-new/docs/graphql/core/actions/types.mdx

```diff
@@ -1,29 +1,27 @@
-<div class="meta" description="Custom GraphQL types for Hasura actions"
-keywords="hasura, docs, actions, custom types">
-
-</div>
+---
+sidebar_label: Custom types
+sidebar_position: 2
+description: Custom GraphQL types for Hasura actions
+keywords:
+  - hasura
+  - docs
+  - actions
+  - custom types
+---
 
 # Custom GraphQL types
 
-<div class="contents" backlinks="none" depth="1" local="">
-
-Table of contents
-
-</div>
-
 ## Introduction
 
 You can add custom GraphQL types in Hasura that you can utilise for
 defining your actions.
 
-<div class="admonition">
-
-Limitations
+:::info Limitations
```

Comment:
same here and any other occurances of Limiations warning etc., - `:::caution Limitations`